src/containers/AuthorsList.js

```diff
@@ -29,35 +29,55 @@ const _ = {
 
 class AuthorsList extends React.Component {
   static fetchData({ store }) {
-    return store.dispatch(fetchNextPageAuthors())
+    return store.dispatch(searchAuthorsIfNeeded(''))
   }
 
   constructor(props) {
     super(props)
+    this.state = {
+      whichAuthorsListToRender: 'authorsList'
+    }
   }
 
   componentDidMount() {
-    const { currentPage, setHeaderInfo, fetchNextPageAuthors } = this.props
-    if (currentPage === (REQUEST_PAGE_START_FROM -1) ) {
-      fetchNextPageAuthors()
-    }
+    const { setHeaderInfo, authorsList, searchAuthorsIfNeeded } = this.props
     setHeaderInfo({
       pageTheme: LIGHT,
       pageType: AUTHORS_LIST,
       pageTitle: PAGE_TITLE
     })
+    if ((this.state.whichAuthorsListToRender==='authorsList') && !authorsList.isFetching && authorsList.currentPage < NUMBER_OF_FIRST_RESPONSE_PAGE) {
```

Comment:
This is to fetch first page of authors list if server rendering was not trigger or didn't get the data.

It will works fine without `(this.state.whichAuthorsListToRender==='authorsList')`. But need to check `authorsList.currentPage < NUMBER_OF_FIRST_RESPONSE_PAGE` to avoid duplicate fetching if server rendering was success.